docs/Frequently Asked Questions/Multiple Process Items/index.md

```diff
@@ -0,0 +1,98 @@
+# Multiple Process Items
+
+The SmartDeviceLink Android library uses multiple process and there are some items that should be understood about why that is necessary and precautions to take while handling that situation. 
+
+## Why does the router service run in its own process?
+
+The router service is designed to live outside the normal lifecycle of the app integrating the SDL framework. The different process allows a level of security to cut off access to the hosting application's data. It also allows the router service to not interfere with the hosting applications runtime; this means if the router service unexpectedly stops or crashes, it will not take down the entire app. This relationship also works in the opposite direction, which is important to maintain a good user experience when apps are connected through a router service.
+
+## Content providers and multi-process issues
+
+Android content providers have a unique lifecycle that does not work in the expected flow. Content providers are actually started before the Application class and following Activities, Services, etc. Some libraries use this to know when their code/module can initialize and always be ready for the entire lifecycle of the application. This is found with many of Google libraries (Firebase, Jetpack, etc).
+
+The issue is that, by default, content providers are only attached to the main process. This process is the same as the application package unless otherwise specified. This means, when the main process starts the content provider will be started, but if a different process is started the content provider will not be started. So if the app has its first start from a component that is designed to run in a different process, the content provider won't be ready by the time those components start up; this includes the application instance for that process. 
+
+### Why is this a problem?
+
+The issue is when there is code in a developer's custom `Application` class that assumes the content provider has already been initialized but an instance of that child `Application` class is created for a process outside the main process.
+
+For example:
+
+```
+public class MyApplication extends Application {
+
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        
+         ModuleUsingContentProviderForInit.doSomething();
+
+    }
+}
+
+```
+
+If an instance is created outside the main process, this application will crash with a runtime exception. 
+
+### Workaround
+
+Depending on the module that uses a content provider for initialization it could be possible to start/initialize it from the `onCreate` method of the extended `Application` class. It should be noted that the module would then need to be setup for a multiple process environment, which is not always the case.
```

Comment:
```suggestion
Depending on the module that uses a content provider for initialization, it could be possible to start/initialize it from the `onCreate` method of the extended `Application` class. It should be noted that the module would then need to be set up for a multiple process environment, which is not always the case.
```